x-pack/plugins/uptime/server/lib/adapters/telemetry/kibana_telemetry_adapter.ts

```diff
@@ -387,6 +410,118 @@ export class KibanaTelemetryAdapter {
     return bucket;
   }
 
+  public static async countOfSyntheticsTests(callCluster: UptimeESClient) {
+    const params = {
+      index: 'synthetics-*',
```

Comment:
Q: Can you test the collector with a user having only the `kibana_system` role? I wonder if `kibana_system` has access to that index. `kibana_system` is the actual user that runs the collection when reporting telemetry.

Reply:
You're right. Thanks for the catch. We have these being called both when a user visits Uptime (who will have the right permissions, in the most likely case), and in the usage collection task. We can remove this call from the usage collection task since `kibana_system` won't have permissions, and keep it only when a user visits Uptime.

Reply:
@afharo We've discussed as a team that we are likely to remove this part of the PR entirely, and handle collecting the data from the Service. It looks like @andrewvc set up a meeting to discuss.